pom.xml

```diff
@@ -67,6 +67,7 @@ under the License.
 		<benchmarkExcludes>org.apache.flink.benchmark.full.*,org.apache.flink.state.benchmark.*,org.apache.flink.scheduler.benchmark.*</benchmarkExcludes>
 		<benchmarks>.*</benchmarks>
 		<executableJava>java</executableJava>
+		<jmhProfArgument> </jmhProfArgument>
```

Comment:
How about adding a property for path for profiling result? And I suggest a relative path under current path for default value.